cloud/src/meta-service/keys.cpp

```diff
@@ -22,18 +22,6 @@
 namespace doris::cloud {
 
 // clang-format off
-// Prefix
-static const char* INSTANCE_KEY_PREFIX = "instance";
```

Comment:
we can do this anther way without moving these constant:
1. adding a function called `get_key_prefixes()` in .cpp
2. extern declare `get_key_prefixes()` in metrics.cpp 

we should keep these key prefixes intact and make it encapsulated inside keys.cpp